app/components/blacklight/gallery/openseadragon_embed_component.rb

```diff
@@ -33,27 +37,26 @@ def render?
       def osd_config
         {
           crossOriginPolicy: false,
-          zoomInButton: "#{id_prefix}-zoom-in",
-          zoomOutButton: "#{id_prefix}-zoom-out",
-          homeButton: "#{id_prefix}-home",
-          fullPageButton: "#{id_prefix}-full-page",
-          nextButton: "#{id_prefix}-next",
-          previousButton: "#{id_prefix}-previous"
+          zoomInButton: "#{@id_prefix}-zoom-in",
+          zoomOutButton: "#{@id_prefix}-zoom-out",
+          homeButton: "#{@id_prefix}-home",
+          fullPageButton: "#{@id_prefix}-full-page",
+          nextButton: "#{@id_prefix}-next",
+          previousButton: "#{@id_prefix}-previous"
         }
       end
 
       def osd_config_referencestrip
         {
           showReferenceStrip: true,
-          referenceStripPosition: 'OUTSIDE',
+          sequenceMode: true,
           referenceStripScroll: 'vertical',
-          referenceStripWidth: 100,
           referenceStripBackgroundColor: 'transparent'
         }
       end
 
-      def id_prefix
-        "osd-#{Blacklight::OpenseadragonHelper.mint_id}".to_param
+      def generate_id_prefix
+        new_id ||= Blacklight::OpenseadragonHelper.mint_id
```

Comment:
Why are you doing the `||=` with a local variable?  Could you instead re-name this method to `id_prefix` and do memoizing with `@id_prefix`?